packages/appstore/src/components/cfds-listing/index.tsx

```diff
@@ -250,6 +251,80 @@ const CFDsListing = () => {
             ) : (
                 <PlatformLoader />
             )}
+
+            {!is_eu_user && (
```

Comment:
@hirad-deriv since we want to show this just for demo, shouldn't we check `!is_real` as well?

Reply:
Done